jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ResourceHandler.java

```diff
@@ -351,6 +358,13 @@ public void setMimeTypes(MimeTypes mimeTypes)
         _mimeTypes = mimeTypes;
     }
 
+    public void setUseFileMapping(boolean useFileMapping)
+    {
+        if (isRunning())
+            throw new IllegalStateException("Unable to set useFileMapping on started " + this.getClass().getSimpleName());
```

Comment:
I'd use `+ this` only because if so happens that users anonymously subclass `ResourceHandler`, the simple name is the empty string, while `toString()` returns a more informative string.

Reply:
Done.